onnxruntime/core/framework/tensor.cc

```diff
@@ -7,6 +7,7 @@
 #include "core/common/safeint.h"
 #include "core/framework/allocatormgr.h"
 #include "core/framework/data_types.h"
+#include "core/framework/ml_value.h"
 #include "core/framework/utils.h"
```

Comment:
nit: I have a dream, that one day we rename this file to match the renamed class it provides. :)